### PR TITLE
Logical operators

### PR DESCRIPTION
**Pull request by avagyan98**

### CODE REVIEW

File: jsLearn/ifelse.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// 1.
+
+let name = prompt("What is the official name of JavaScript?", 'JS');
+
+if(name == "ECMAScript") {
+    alert("True");
+}else {
+    alert("Do not know? ECMAScript!");
+}
```
Comment (robhov97) [inline jsLearn/ifelse.js]: chishty asa, artagrel es?

File: jsLearn/ifelse.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+let resalt = (a + b < 4) ? "few" : "many";
+
+
+// 4.
+
+let massige = login == "Employee"
+    ? "Hey" : login == "Director"
+    ? "Hello" : login == ''
+    ? "No login" : "";
+
```
Comment (robhov97) [inline jsLearn/ifelse.js]: artagrel es?

File: jsLearn/logicalOperators.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+let age = prompt("write age", 20);
+let result = age >= 14 && age <= 90 ? true : false;
+
+
+// 2
+
+let age = prompt("write age", 20);
+let result = !(age >= 14 && age <= 90) ? true : false;
+let result1 = age < 14 && age >90 ? true : false;
```
Comment (robhov97) [inline jsLearn/logicalOperators.js]: both your conditions return a boolean value, no need for ternary operator

File: jsLearn/logicalOperators.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+
+// 3.
+
+let login = prompt("Who is this?", "name");
+
+if(login == "Admin"){
+    let parol = prompt("Parol", "");
+
+    if(parol == "I'm in charge") {
```
Comment (robhov97) [inline jsLearn/logicalOperators.js]: xi brat?

**Review by robhov97 — CHANGES_REQUESTED** [review]

File: jsLearn/ifelse.js (hunk 1 above)
Comment (avagyan98) [reply jsLearn/ifelse.js]: der patasxanner@ chem bacel

File: jsLearn/ifelse.js (hunk 2 above)
Comment (avagyan98) [reply jsLearn/ifelse.js]: chm el googlel)

File: jsLearn/logicalOperators.js (hunk 3 above)
Comment (avagyan98) [reply jsLearn/logicalOperators.js]: done

File: jsLearn/logicalOperators.js (hunk 4 above)
Comment (avagyan98) [reply jsLearn/logicalOperators.js]: inch@ xi, chhaskaca chishtn asac)

Comment (robhov97) [reply jsLearn/logicalOperators.js]: 2 havasary

File: jsLearn/ifelse.js (hunk 2 above)
Comment (robhov97) [reply jsLearn/ifelse.js]: ba xi es sax texery 2 havasar drel? ham mer proyektum chenq ogtagortsum, hamel en ory hatuk asi vor es 2 havasarov ashxati code chgres. Isk grqi meji tnayinum 2 havasarov er arats :D

**Review by robhov97 — CHANGES_REQUESTED** [review]
replace 2 equals with 3 equals when comparing

File: jsLearn/ifelse.js (hunk 2 above)
Comment (avagyan98) [reply jsLearn/ifelse.js]: moracel ei vor asel es, de teman kardaci nor greci tenc 2 hatov er mejs tpavorvac arag greci, che toshni ban em asum vochmi ban chem nayel

File: jsLearn/logicalOperators.js (hunk 4 above)
Comment (avagyan98) [reply jsLearn/logicalOperators.js]: poxaca, mi  2 tnayin el uxarki

File: jsLearn/ifelse.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+if(name === "ECMAScript") {
+    alert("True");
+}else {
+    alert("Do not know? ECMAScript!");
+}
+
+
+// 2.
+
+let number = prompt("write number", 100);
```
Comment (robhov97) [inline jsLearn/ifelse.js]: prompt returns a string, convert it's returned value into number for safety

Comment (robhov97) [reply jsLearn/ifelse.js]: use const if you not intend to change the value of the variable

File: jsLearn/ifelse.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
+// 1.
+
+let name = prompt("What is the official name of JavaScript?", 'JS');
```
Comment (robhov97) [inline jsLearn/ifelse.js]: const

File: jsLearn/ifelse.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+}else if(number < 0) {
+    alert(-1)
+}else {
+    alert(0)
+}
+
+
+// 3.
+
+let resalt = (a + b < 4) ? "few" : "many";
```
Comment (robhov97) [inline jsLearn/ifelse.js]: typo and const 
```suggestion
const result = (a + b < 4) ? "few" : "many";
```

File: jsLearn/ifelse.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+
+// 4.
+
+let massige = login === "Employee"
```
Comment (robhov97) [inline jsLearn/ifelse.js]: typo and const
```suggestion
const massage = login === "Employee"
```

File: jsLearn/logicalOperators.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+// 1.
+
+let age = prompt("write age", 20);
+let result = age >= 14 && age <= 90;
```
Comment (robhov97) [inline jsLearn/logicalOperators.js]: const

File: jsLearn/logicalOperators.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+
+// 2
+
+let age = prompt("write age", 20);
```
Comment (robhov97) [inline jsLearn/logicalOperators.js]: double declaration of age

Comment (robhov97) [reply jsLearn/logicalOperators.js]: age is string but used as a number, convert the returned value of prompt into number for safety

**Review by robhov97 — CHANGES_REQUESTED** [review]

File: jsLearn/ifelse.js (hunk 5 above)
Comment (avagyan98) [reply jsLearn/ifelse.js]: done

File: jsLearn/ifelse.js (hunk 6 above)
Comment (avagyan98) [reply jsLearn/ifelse.js]: done

File: jsLearn/ifelse.js (hunk 7 above)
Comment (avagyan98) [reply jsLearn/ifelse.js]: done

File: jsLearn/logicalOperators.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
+// 1.
+
+const age = prompt("write age", 20);
+const result = age >= 14 && age <= 90;
+
+
+// 2
+
+const age = Number(prompt("write age", 20));
```
Comment (robhov97) [inline jsLearn/logicalOperators.js]: double declaration

**Review by robhov97 — CHANGES_REQUESTED** [review]

**Review by robhov97 — APPROVED** [review]

**Pull request merged**